### PR TITLE
RF POM AT_12.03_007 Rename freestyle project

### PR DESCRIPTION
**Pull request by LanaBondarenko**
https://trello.com/c/5QaOOU6S/1562-rf-pom-at1203007-freestyle-project-rename-project-using-side-menu-on-project-page

### CODE REVIEW

File: cypress/pageObjects/FreestyleProjectConfigurePage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import FreestyleProjectPage from "./FreestyleProjectPage";
+
+class FreestyleProjectConfigurePage {
+    getProjectConfigSaveBtn = () => cy.get('button[name=Submit]');
+    
+    clickSaveBtnAndGoFreestyleProjectPage() {
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectConfigurePage.js:6]: Please rename to `clickSaveBtnAndGoFreestyleProject()`

File: cypress/pageObjects/FreestyleProjectConfigurePage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        this.getProjectConfigSaveBtn().click();
+        return new FreestyleProjectPage;
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectConfigurePage.js:8]: add please `()`
`return new FreestyleProjectPage();`

File: cypress/e2e/tests/freestyleProject.cy.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            .clickNewItemSideMenuLink()
+            .typeNewItemNameInputField(newItemData.freestyleProjectName)
+            .selectFreestyleProjectItem()            
+            .clickOkBtnAndGoFreestyleProjectConfig()
+            .clickSaveBtnAndGoFreestyleProjectPage()
+            .selectRenameProjectMenu ()
+            .typeNewNameInputFild (freestyleProjectNewName)
+            .clickRenameBtn ()
+            .getProjectName ()            
+            .should('have.text', "Project " + freestyleProjectNewName)
```
Comment (Rom-T) [inline cypress/e2e/tests/freestyleProject.cy.js:22]: Please move `"Project "` to the fixtures

File: cypress/pageObjects/FreestyleProjectPage.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import FreestyleProjectRenamePage from "./FreestyleProjectRenamePage";
+
+class FreestyleProjectPage {
+    getRenameProjectMenu = () => cy.get('#side-panel a[href$="rename"]');
+    getProjectName = () => cy.get('.job-index-headline');
+    
+    selectRenameProjectMenu () {
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectPage.js:7]: Please rename to `clickRenameSideMenuLink`

File: cypress/pageObjects/FreestyleProjectPage.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import FreestyleProjectRenamePage from "./FreestyleProjectRenamePage";
+
+class FreestyleProjectPage {
+    getRenameProjectMenu = () => cy.get('#side-panel a[href$="rename"]');
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectPage.js:4]: Please rename to `getRenameSideMenuLink`

File: cypress/pageObjects/FreestyleProjectPage.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    getProjectName = () => cy.get('.job-index-headline');
+    
+    selectRenameProjectMenu () {
+        this.getRenameProjectMenu().click();
+        return new FreestyleProjectRenamePage;
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectPage.js:9]: add please `()`
`return new FreestyleProjectRenamePage();`

File: cypress/pageObjects/FreestyleProjectPage.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    }    
+}
+
+export default FreestyleProjectPage;
+
+
+
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectPage.js:16]: Please delete 2 empty lines. Leave only 1 blank line at the end

File: cypress/pageObjects/FreestyleProjectRenamePage.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    getRenameBtn = () => cy.get('button[name=Submit]');
+
+    typeNewNameInputFild (name) {
+        this.getNewNameInputFild().clear().type(name);
+        return this;
+    }
+
+    clickRenameBtn () {
+        this.getRenameBtn().click();
+        return new FreestyleProjectPage;
```
Comment (Rom-T) [inline cypress/pageObjects/FreestyleProjectRenamePage.js:14]: add please `()`
return new `FreestyleProjectPage();`

File: cypress/pageObjects/NewItemPage.js [hunk 9]
```diff
@@ -27,6 +29,16 @@ class NewItemPage {
...
     };
+
+    selectFreestyleProjectItem() {
+        this.getFreestyleProjectItem().click();
+        return this;
+    };
+
+    clickOkBtnAndGoFreestyleProjectConfig() {
+        this.getNewItemOkBtn().click();
+        return new FreestyleProjectConfigurePage;
```
Comment (Rom-T) [inline cypress/pageObjects/NewItemPage.js:40]: add please `()`
`return new FreestyleProjectConfigurePage();`

File: cypress/e2e/tests/freestyleProject.cy.js [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+        homePage
+            .clickNewItemSideMenuLink()
+            .typeNewItemNameInputField(newItemData.freestyleProjectName)
+            .selectFreestyleProjectItem()            
+            .clickOkBtnAndGoFreestyleProjectConfig()
+            .clickSaveBtnAndGoFreestyleProjectPage()
+            .selectRenameProjectMenu ()
+            .typeNewNameInputFild (freestyleProjectNewName)
+            .clickRenameBtn ()
+            .getProjectName ()
```
Comment (Rom-T) [inline cypress/e2e/tests/freestyleProject.cy.js:21]: Please remove spaces between methods and brackets

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by IrynaKolh — APPROVED** [review]
Looks great!

**Pull request merged**